src/components/VideoCards.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import Checkmark from './Checkmark';
+import ProgressBar from './ProgressBar';
+import { percentageTime } from '../util/time';
+
+const lecturePropTypes = {
+  title: PropTypes.string,
+  cover: PropTypes.string,
+  watched: PropTypes.bool,
+  position: PropTypes.string, // time watched until
+  length: PropTypes.string,
+};
+
+const CheckWrapper = styled.div`
+  position: absolute;
+  top: 10px;
+  right: 10px;
+`;
+
+const ProgressBarWrapper = styled.div`
+  margin-top: auto;
+`;
+
+const Video = styled.div`
+  background-color: #c4c4c4;
+  position: relative;
+  height: 300px;
+  width: 100%;
+  border-radius: 10px;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+`;
+
+const VideoTitle = styled.div`
+  width: 100%;
+  text-align: center;
+  margin-top: 0.5rem;
+`;
+
+// eslint-disable-next-line
+const VideoCard = ({ title, cover, watched, position, length }) => (
+  <div>
+    <Video>
+      {watched && (
+        <CheckWrapper>
+          <Checkmark />
+        </CheckWrapper>
+      )}
+      <ProgressBarWrapper>
+        <ProgressBar percent={percentageTime(position, length)} height={12} />
+      </ProgressBarWrapper>
+    </Video>
+    <VideoTitle>{title}</VideoTitle>
+  </div>
+);
+
+VideoCard.propTypes = lecturePropTypes;
+
+VideoCard.defaultProps = {
+  title: 'Lecture ...',
+  cover: null,
+  watched: false,
+  position: '10:23',
+  length: '40:33',
+};
+
+const VideoContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 3rem 6rem;
+  margin: ${props => props.m || 0};
+`;
+
+const VideoCards = ({ lectures }) => (
+  <VideoContainer m='3rem 0'>
+    {lectures.reverse().map(lecture => (
```

Comment:
`reverse` mutates the array, which probably isn't intended. Make a shallow copy first to avoid mutating the array.
```suggestion
    {[...lectures].reverse().map(lecture => (
```